sql/2019/04_Media/04_03.sql

```diff
@@ -1,19 +1,26 @@
 #standardSQL
 # 04_03: SVG usage
 SELECT
-  client,
   percentile,
+  client,
   APPROX_QUANTILES(requests, 1000)[OFFSET(percentile * 10)] AS requests,
   ROUND(APPROX_QUANTILES(bytes, 1000)[OFFSET(percentile * 10)] / 1024, 2) AS kbytes
 FROM (
-  SELECT _TABLE_SUFFIX AS client, COUNT(0) AS requests, SUM(respSize) AS bytes
-  FROM `httparchive.summary_requests.2019_07_01_*`
-  WHERE format = 'svg'
-  GROUP BY client, pageid),
+  SELECT
+    _TABLE_SUFFIX AS client,
+    COUNT(0) AS requests,
+    SUM(respSize) AS bytes
+  FROM
+    `httparchive.summary_requests.2019_07_01_*`
+  WHERE
+    format = 'svg'
```

Comment:
1. Can we add a % of pages with svg requests?
2. What about a 4_03b where we check for the % of pages that use svgs at all (combining 4_04s check of inline). See what the adoption/usage rate is.

Reply:
Added 03b with both suggestions.